README.md

```diff
@@ -22,6 +22,81 @@ Or relevant sources pages, to get started head to:
 
 ## Quickstart
 
+The fastest (less than 20min) way to build z-wave-protocol-controller from scratch
+is to delegate all tasks to docker.
+
+This one-liner will do download latest release, setup environment, build, test, package...:
+
+```sh
+docker build https://github.com/SiliconLabsSoftware/z-wave-protocol-controller.git#ver_1.7.0
+```
+
+Power users might prefer to work in sources tree on a native GNU/Linux environment.
+
+The project is CMake based,
+to prepare the environment, have a look at [./helper.mk](helper.mk)'s  details 
+for needed steps to setup developer system before using CMake normally.
+
+At the moment stable version of Debian is supported,
+so it should work also in relatives projects (Ubuntu, RaspiOS, WSL2 etc)
+and should be easy to adapt to other distributions.
+
+```sh
+./helper.mk help
+./helper.mk setup  # To setup developer system (once)
+./helper.mk VERBOSE=1 # Default build tasks verbosely (depends on setup)"
+./helper.mk run # Run entry-point application
+```
+
+It should print zpc's help.
+
+To use it, a Silicon Labs' Z-Wave NCP should be plugged in USB port
+to verify you can check firmware version:
+
+```sh
+serial=$(ls /dev/serial/by-id/usb-Silicon_Labs* | head -n1)
+./helper.mk all run run_args="--zpc.serial=${serial} --zpc.ncp_version"
+# <i> [zpc_ncp_update] chip_serial_api_version: 7.23.1
+```
+
+Then let's interact with ZPC's inbuilt shell without installing it.
+
+```sh
+serial=$(ls /dev/serial/by-id/usb-Silicon_Labs* | head -n1)
+run_args="--zpc.serial=${serial}"
+mapdir="applications/zpc/components/dotdot_mapper/rules"
+run_args="$run_args --mapdir=${mapdir}"
+datastore_file="tmp.db"
+run_args="$run_args --zpc.datastore_file=${datastore_file}"
+cache_path="tmp"
+run_args="$run_args --zpc.ota.cache_path=${cache_path}"
+
+sudo apt install -y mosquitto # Is a strong runtime dependency
+mkdir -p ${cache_path}
+./helper.mk run run_args="$run_args"
+
+ZPC>help
+==================================================
+Unify Command line interface Help:
+==================================================
+(...)
+exit  :Exit the application
+(...)
+zwave_home_id Print Z-Wave Home ID
+(...)
+zwave_add_node  :Add a Z-Wave node to the network
+(...)
+zwave_set_default Reset Z-Wave network
+(...)
+ZPC> zwave_home_id
+Z-Wave Home ID:
+1BADC0DE
+ZPC> zwave_add_node
+(...)
+```
+
+Refer to [./doc](doc) for more (using MQTT, WebApp etc).
```

Comment:
Please double check dev-ui is explained there...

to make long story short you need to download it from unify repo
One you installed the deb file
you can check installed services using
sudo systemctl
and then restart it if needed
sudo systemctl restart uic-dev-ui  something